managers/bosh-manager/DirectorService.js

```diff
@@ -232,22 +231,13 @@ class DirectorService extends BaseDirectorService {
     const operation = {
       type: 'update'
     };
-    // service fabrik operation token
-    const token = _.get(params.parameters, 'service-fabrik-operation', null);
-    if (token) {
-      _.unset(params.parameters, 'service-fabrik-operation');
-      _.set(params, 'scheduled', true);
-    }
     return this
       .initialize(operation)
-      .then(() => token ? jwt.verify(token, config.password) : null)
-      .then(serviceFabrikOperation => {
-        logger.info('SF Operation input:', serviceFabrikOperation);
-        this.operation = _.get(serviceFabrikOperation, 'name', 'update');
-        // normal update operation
+      .then(() => {
+        logger.info('Parameters for update operation:', _.get(params, 'parameters'));
+        this.operation = this.operation || 'update';
```

Comment:
Is this line required?

Reply:
yes. for directly invoking instance and call. Breaks UT otherwise.